fastapi_endpoint.py

```
@@ -0,0 +1,127 @@
+app = FastAPI()
+
+
+def copy_folder(source_folder, destination_folder):
+    try:
+
+        if not os.path.exists(source_folder):
+            raise FileNotFoundError(f"Source folder does not exist: {source_folder}")
+        shutil.copytree(source_folder, os.path.join(destination_folder, os.path.basename(source_folder)),dirs_exist_ok=True)
+        logging.info(f"Successfully copied {source_folder} to {destination_folder}")
+
+    except FileNotFoundError as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)